rs/https_outcalls/adapter/tests/server_test.rs

```diff
@@ -460,6 +470,7 @@ MIGHAgEAMBMGByqGSM49AgEGCCqGSM49AwEHBG0wawIBAQQgob29X4H4m2XOkSZE
             body: "hello".to_string().as_bytes().to_vec(),
             max_response_size_bytes: 512,
             socks_proxy_allowed: false,
+            socks_proxy_addrs: vec![],
```

Comment:
can we use ..Default::default() ?

This way we can remove same default values and in the future when we want to add another field we don't need to change all the tests.

Reply:
done